flink-python/apache-flink-libraries/setup.py

```diff
@@ -98,8 +98,12 @@ def find_file_path(pattern):
                   file=sys.stderr)
             sys.exit(-1)
         flink_version = VERSION.replace(".dev0", "-SNAPSHOT")
-        FLINK_HOME = os.path.abspath(
-            "../../flink-dist/target/flink-%s-bin/flink-%s" % (flink_version, flink_version))
+        flink_homes = glob.glob('../../flink-dist/target/flink-*-bin/flink-*')
+        if len(flink_homes) != 1:
+            print("More than one Flink home directory found {0}".format(flink_homes),
```

Comment:
Is it possible to get to this line if we did not package Flink yet? Even if not, I could end up cleaning up the dist target dir, so there is a possibility that `len(flink_homes)` will be `0` and in that case the "More than one..." error msg. is misleading. I think this can be rephrased in a more general way, maybe "Ambiguous Flink home directory found: ..." or something.

Reply:
Good point, tried it out only w/ more than one dir. Changing...